server/core/store.go

```diff
@@ -171,11 +178,16 @@ func (s *StoreInfo) GetUptime() time.Duration {
 	return 0
 }
 
-const defaultStoreDownTime = time.Minute
+// If a store's last heartbeat is before now-storeDisconnectDuration, the store
```

Comment:
If hard to read, may be:
"If a store's last heartbeat is storeDisconnectDuration ago, ...."